src/releng_docs/shipit_signoffs

```diff
@@ -0,0 +1 @@
+../shipit_signoff/docs
```

Comment:
To your first point, livereload works fine if you add a second watch directory.

And on "git log" - it breaks because it doesn't follow symlinks. For example, here is what happens in a test repo:
```
➜  foo git:(master) ls -lR releng_docs 
releng_docs:
total 4
drwxrwxr-x 2 bhearsum bhearsum 4096 Apr 19 09:13 shipit_signoffs

releng_docs/shipit_signoffs:
total 4
-rw-rw-r-- 1 bhearsum bhearsum 13 Apr 19 09:13 index.html
➜  foo git:(master) ls -lR shipit_signoffs 
shipit_signoffs:
total 0
lrwxrwxrwx 1 bhearsum bhearsum 30 Apr 19 09:12 docs -> ../releng_docs/shipit_signoffs
➜  foo git:(master) git log | cat
commit 01cdc84049d34783466822f70ebc78a262311a4f (HEAD -> master)
Author: Ben Hearsum <bhearsum@mozilla.com>
Date:   Wed Apr 19 09:13:29 2017 -0400

    Another change

commit 82425e9dfc29bf22da2e98e2400014ff32cd1069
Author: Ben Hearsum <bhearsum@mozilla.com>
Date:   Wed Apr 19 09:13:08 2017 -0400

    A change!

commit a3d84fd599d1cd0ca3269f0905ae5406413b1838
Author: Ben Hearsum <bhearsum@mozilla.com>
Date:   Wed Apr 19 09:12:46 2017 -0400

    Commit initial docs.
➜  foo git:(master) git log releng_docs | cat 
commit 01cdc84049d34783466822f70ebc78a262311a4f (HEAD -> master)
Author: Ben Hearsum <bhearsum@mozilla.com>
Date:   Wed Apr 19 09:13:29 2017 -0400

    Another change

commit 82425e9dfc29bf22da2e98e2400014ff32cd1069
Author: Ben Hearsum <bhearsum@mozilla.com>
Date:   Wed Apr 19 09:13:08 2017 -0400

    A change!

commit a3d84fd599d1cd0ca3269f0905ae5406413b1838
Author: Ben Hearsum <bhearsum@mozilla.com>
Date:   Wed Apr 19 09:12:46 2017 -0400

    Commit initial docs.
➜  foo git:(master) git log shipit_signoffs | cat
commit a3d84fd599d1cd0ca3269f0905ae5406413b1838
Author: Ben Hearsum <bhearsum@mozilla.com>
Date:   Wed Apr 19 09:12:46 2017 -0400

    Commit initial docs.
```